test/t8_schemes/t8_gtest_nca.cxx

```diff
@@ -170,50 +161,72 @@ t8_recursive_nca_check(t8_element_t *check_nca, t8_element_t *desc_a,
     int level_b = ts->t8_element_level(parent_b);
     int num_children_a, num_children_b;
     int i, j;
-    /* If one parent has reached the maximal level, the test returns*/
+    /* If one parent has reached the maximal level, the test returns */
     if(level_a == max_lvl || level_b == max_lvl){
```

Comment:
Nitpick: spaces between if/for/while and opening parenthesis.

Reply:
Indenting was off for this part of the code. I turned it on for this part